_automating-configurations/api/create-workflow.md

```diff
@@ -53,6 +53,15 @@ When set to `true`, the [Provision Workflow API]({{site.url}}{{site.baseurl}}/au
 
 By default, workflows are validated when they are created to ensure that the syntax is valid and that the graph does not contain cycles. This behavior can be controlled with the `validation` query parameter. If `validation` is set to `all`, OpenSearch performs a complete template validation. Any other value of the `validation` parameter suppresses validation, allowing an incomplete/work-in-progress template to be saved. To disable template validation, set `validation` to `none`:
 
+You can not update a full workflow that has been provisioned, but you can update fields other than the `workflows` field, such as the name and description.
+
+```json
+PUT /_plugins/_flow_framework/workflow/<workflow_id>?update_fields=true
```

Comment:
Is this a complete request? Do you need a request body that specifies which fields are to be updated?